<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatingValveSettings</class>
 <widget class="QWidget" name="HeatingValveSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heating Valve Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QWidget" name="checkboxes" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>300</width>
     <height>190</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="enableManualPosition">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>260</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use manual valve position</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableSummerMode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Summer mode</string>
    </property>
   </widget>
   <widget class="QSlider" name="valvePositionSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="raise">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>125</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lower">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>125</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="temperatureValue">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>125</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>17.000000000000000</double>
    </property>
    <property name="maximum">
     <double>25.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>21.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valvePositionValue">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>55</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelDesired">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>125</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Temperature:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="learnValve">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Learn Heating Valve</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string notr="true">21,0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string notr="true"> Â°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisplayedValve">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
